### DISCUSSION TITLE
Support for other Linux based operating systems (raspberry pi)

### DISCUSSION THREAD

**Discussion by muendlein** [mworion/MountWizzard4#120] [Closed Feature requests] [answered]
As of right now it seems like only Ubuntu is supported. Are there any plans to support i.e. Raspbian?

**Comment by mworion** [comment]
Hi,
as MW4 uses python 3.7-3.9 in a virtual environment, the only topic is to take care, that this is available. You have to take care of precompiled libraries or an environment with a compiler available for packages based on C(++). 
I only have ubuntu as virtual test environment. If you made it happen, please feel free to report about your way how to do it. I added some assets for raspi4 with ubuntu, but this does not scale up for me, but it should be not too complicated for a user to do.
Michel

**Comment by mworion** [comment, marked as answer]
Astroberry and stellar mate is supported from v2.1.4
